DeusEx/Classes/DXRFixup.uc

```diff
@@ -0,0 +1,154 @@
+class DXRFixup expands DXRBase;
+
+function FirstEntry()
+{
+    local Actor a;
+    local ScriptedPawn p;
+
+    Super.FirstEntry();
+    
+    switch(dxr.localURL)
+    {
+        case "06_HONGKONG_TONGBASE":
+            foreach AllActors(class'Actor', a)
+            {
+                switch(string(a.Tag))
+                {
+                    case "AlexGone":
+                    case "TracerGone":
+                    case "JaimeGone":
+                        a.Destroy();
+                        break;
+                    case "Breakintocompound":
+                    case "LumpathPissed":
+                        Trigger(a).bTriggerOnceOnly = False;
+                        break;
+                    default:
+                        break;
+                }
+            }    
+            break;
+        case "06_HONGKONG_WANCHAI_MARKET":
+            foreach AllActors(class'Actor', a)
+            {
+                switch(string(a.Tag))
+                {
+                    case "DummyKeypad01":
+                        a.Destroy();
+                        break;
+                    case "BasementKeypad":
+                    case "GateKeypad":
+                        a.bHidden=False;
+                        break;    
+                    case "Breakintocompound":
+                    case "LumpathPissed":
+                        Trigger(a).bTriggerOnceOnly = False;
+                        break;
+                    default:
+                        break;
+                }
+            }
+            
+            break;
+        default:
+            break;
+    }
+}
+
+function AnyEntry()
+{
+    Super.AnyEntry();
+    
+    doFixup();
+    
+}
+
+function ReEntry()
+{
+    Super.ReEntry();
+    doFixup();
+}
+
+function doFixup()
+{
+    local Actor a;
+    local ScriptedPawn p;
+    local name flagName;
+    local bool boolFlag;
+    
+    switch(dxr.localURL)
+    {
+        case "06_HONGKONG_TONGBASE":
+            flagName = dxr.Player.rootWindow.StringToName("QuickLetPlayerIn");
```

Comment:
For whatever reason, GetBool doesn't accept a string as input (Unlike GetInt, since I saw you were directly using a string), only a name, so the conversion is necessary.  I can change the double quotes to single quotes though for sure.

Reply:
let me rephrase
`"QuickLetPlayerIn"` is a string because of double quotes
`'QuickLetPlayerIn'` is a name because of single quotes
it's a strange oddity of UnrealScript lol

Reply:
Ohhhhhhhh, that is "fun", or perhaps 'fun'